src/theme/ReferenceCodeBlock/index.tsx

```diff
@@ -111,7 +118,7 @@ function ReferenceCode(props: ReferenceCodeBlockProps) {
 
     const customProps = {
         ...props,
-        metastring: ` title="${codeSnippetDetails.title}"`,
+        metastring: getTitle(props, codeSnippetDetails),
```

Comment:
I don't think we need an extra function for this.
```suggestion
        metastring: props.title ? ` title="${props.title.replace('"', '')}"` : ` title="${codeSnippetDetails.title}"`,
```